src/pages/ProductInner/ProductInner.scss

```diff
@@ -0,0 +1,71 @@
+.ProductInner {
+
+    margin-top: 40px;
+    margin-bottom: 80px;
+
+    &__link {
+        display: inline-block; 
+        font-weight: 400;
+        font-size: 16px;
+        line-height: 24px;
+        text-transform: uppercase;
+        margin-bottom: 25px;
+        color: #fff; 
+        background:#022B32;
+        padding: 3px 10px;
+        border-radius: 6px; 
+    }
+
+    &__content {
+        display: flex;
+        flex-wrap: wrap;
+    }
+
+        
+
+    &__image-container {
+        position: relative;
+        padding: 0 0 24% 0;
+        max-width: 33%;
+        width: 100%;
+
+        @media (max-width: 480px) {
+            max-width: 100%;
+            padding: 0 0 100% 0;
+        }
+
+        img {
```

Comment:
If you are using BEM, try to avoid accessing element by tag name

Use &__image class instead